conf/k8s-authn-webhook-config

```diff
@@ -0,0 +1,17 @@
+clusters:
+- name: authn-webhook
```

Comment:
@charandas for the custom storage solution, we can remove the authnz webhooks correct?

or maybe just the authz one 🤔

Reply:
Yes, we can remove the `authz` webhook.